build.gradle

```diff
@@ -1,83 +1,30 @@
+apply from: 'scripts/gradle/gradle-commons.gradle'
+apply from: 'scripts/gradle/zendesk-repos.gradle'
+
 ext {
-    compileSdkVersion = 23
-    buildToolsVersion = "23.0.3"
-    minSdkVersion = 9
-    targetSdkVersion = 23
-    supportLibVersion = "23.4.0"
-    versionName = "1.3.0.1-SNAPSHOT"
+    compileSdkVersion = 25
+    buildToolsVersion = "25.0.2"
 
-    // Read from environment, fallback to ~/.gradle/gradle.properties. Required for snapshots
-    repoUsername = getConfigurationParameter("zendeskOssRepoUsername")
-    repoPassword = getConfigurationParameter("zendeskOssRepoPassword")
-}
+    minSdkVersionCore = 10
+    minSdkVersionUi = 14
 
-/**
- * Looks for the specified configuration key in this order:
- *
- * <ol>
- *     <li>system environment</li>
- *     <li>project properties</li>
- * </ol>
- *
- * If it isn't found, then the empty string is returned. This stops the gradlew from failing until
- * it actually needs those values.
- *
- * @param keyName The key to look for
- * @return The value or the empty string if not found
- */
-def getConfigurationParameter(String keyName) {
-    return System.getenv(keyName) ?:
-            rootProject.hasProperty(keyName) ? rootProject.properties[keyName] : ''
-}
+    targetSdkVersion = 25
 
-/**
- * Checks if we're building in a test mode, false otherwise
- * <p>
- *     Test mode is based on the existence of an environmental variable called zdRunTestSdk
- * <p>
- *
- * @return if we're building in a test mode, false otherwise
- */
-def buildingInTestMode() {
-    return "$System.env.TEST_SDK" != "null"
-}
+    supportLibVersion = "25.1.0"
+    versionName = "2.0.0.1-SNAPSHOT"
 
-/**
- * Checks if we're building on CI, false otherwise
- *
- * <p>
- *     Test mode is based on the existence of an environmental variable called CI. This is set
- *     by the CI environment, not us.
- * <p>
- *
- * @return true if we're building on CI, false otherwise
- */
-def buildingOnCi() {
-    "$System.env.CI" != "null"
+    buildSettings = [
+            localBuild : true,
```

Comment:
Yes, the default should be `true`. 
There not many situations where this flag has to be `false`. The only one I can think of is releasing artifacts (snapshot/release). For doing that travis has the power to override this flag. https://github.com/zendesk/belvedere/blob/schlan/project_v2.0/scripts/gradle/gradle-commons.gradle#L37-L43